server/api/game/controller.js

```diff
@@ -61,11 +63,23 @@ class Game {
 
   endGame = () => {
     clearInterval(this.gameState.interval);
+    this.gameState.status = 2;
   };
 
-  moveGhosts = ({ ghosts, gridState, scatterStart }) => {
+  ifPacmanOnGhost = (ghost) => {
+    const { players } = this.gameState;
+    const { x, y } = ghost;
+    const canConsumeGhost = Object.values(players)
```

Comment:
Instead of using .find, you can use .some as it automatically evaluates to true/false